frontend/src/features/mapSideBar/components/MapSideBarLayout.tsx

```diff
@@ -64,7 +69,10 @@ const MapSideBarLayout: React.FunctionComponent<IMapSideBarLayoutProps> = ({
               <Title className="mr-auto">{title}</Title>
               {!hidePolicy && <InventoryPolicy />}
               <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
-                <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
+                <CloseIcon
+                  title="close"
+                  onClick={() => setShowSideBar(false, undefined, undefined, true)}
```

Comment:
what's the difference here if you're just setting things to undefined anyway?

Reply:
previously was just sending false to the first param, now also sending true to the last param.